frontend/src/app/component/home/home.component.html

```diff
@@ -0,0 +1,45 @@
+<!-- <app-sidebar></app-sidebar> -->
+<app-navbar></app-navbar>
+
+<div class="button-container">
+  <!-- <h1>Add more users</h1> -->
+  <button routerLink="/form" class="btn btn-success" style="margin-left: 10px">Add User</button>
```

Comment:
Don't use inline CSS ever Its not good practice. use classes to give custom CSS or use Bootstrap to apply already defined classes.